src/Interface/Others/Report/Results/EncounterStats.js

```diff
@@ -218,11 +219,13 @@ class EncounterStats extends React.PureComponent {
         </div>
       );
     }
-
+    // If there are below 100 parses for a given spec, use this amount to divide with to get accurate percentages.
+    // This also enables us to work around certain logs being anonymised - as this will then ignore those, and cause us to divide by 99, making our percentages accurate again.
+    this.amountOfParses = Object.values(this.state.mostUsedTalents[0]).reduce((a, b) => a + b);
```

Comment:
Reduce should always have a default value, and please use descriptive variable names. `a` and `b` are unclear.

Reply:
Please put magic numbers (0) in constants at the top